### ISSUE TITLE
Question about camera pose

### ISSUE THREAD

**Issue by zjwfufu** [pcl3dv/PSAvatar#6]
Hello!
Thanks for the great work! 
I note that in previous issues you said you were using the preprocessing pipeline developed by IMAvatar/PointAvatar, and in the first version on arxiv, a table in supplementary material lists the learning rate of the parameter to be optimized has one member named "camera". 
So I wonder if how you optimize the camera parameter in your implementation. The original 3DGS implementation does not pass the gradient of the camera parameter, can you share how you did it? Or did you borrow pytorch3d's rasterizer to optimize the camera parameters during rendering with point-splat?

**Comment by pcl3dv** [comment]
Hi, Thanks for your attention! In a similar way as PointAvatar, we optimize the camera parameters while determining the shape model.
